analyses/doublet-detection/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Base image on the Bioconductor 3.19 image
+FROM bioconductor/r-ver:3.19
```

Comment:
Just noting that I discovered that this image unfortunately does not have `pandoc` installed. I recommend adding the following somewhere in here to install it (probably after labels, before renv):

```
RUN apt-get -y update &&  \
    DEBIAN_FRONTEND=noninteractive \
    apt-get install --no-install-recommends -y \
    pandoc \
    && rm -rf /var/lib/apt/lists/*
```